ci/pipeline.yml

```diff
@@ -190,9 +311,26 @@ resources:
     type: git
     icon: github
     source:
-      uri: https://github.com/18F/federalist-proxy
+      uri: ((proxy-repository-url))
```

Comment:
Thoughts on using env variable so we do not need to make another commit when migrating from 18F to cloud-gov org?

Reply:
Nice, good thinking!